docs/howto/custom-management-commands.txt

```diff
@@ -18,9 +18,7 @@ directory whose name doesn't begin with an underscore. For example::
         __init__.py
```

Comment:
@timgraham I know it has been a long time since comitted. why this `__init__.py` remains though the other __init__ files were deleted?

Reply:
This file isn't in a  management command directory.  There was [a concern](https://code.djangoproject.com/ticket/23919#comment:102) with removing all `__init__.py` files.